src/Browscap.php

```diff
@@ -1,6 +1,14 @@
 <?php
-declare(strict_types = 1);
+/**
```

Comment:
We really don't need a comment for every file :/

Reply:
fixed